todo_app/tests/integration/integration_test.py

```diff
@@ -6,7 +6,7 @@
 
 @pytest.fixture
 def client():
-    file_path = find_dotenv('.env.test')
+    file_path = find_dotenv('.env.template')
```

Comment:
I guess this works, but it feels a little odd that we're using the template for two distinct tasks (placeholders for new devs, and also placeholders for tests). As a user, I'd be surprised if I changed one of the placeholder values and then broke the tests